corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -363,7 +363,7 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             if (!(
                 e.target.classList.contains('module-case-list-column-checkbox') ||  // multiselect checkbox
                 e.target.classList.contains("select-row-checkbox") ||               // multiselect select all
-                $(e.target).is('a') ||                                              // actual link, as in markdown
+                $(e.target).parent().is('a') ||                                     // actual link, as in markdown
```

Comment:
What if this were `$(e.target).closest('a').length`? Then it could handle both links and their descendants (if we removed the underlining, this code wouldn't need to change again).

Reply:
If I'm reading docs right, `.closest()` traverses up the DOM tree until it finds a matching element (or no matching elements). Wouldn't that add unnecessary overhead for clicks that aren't on a link? I think my preference would be to keep both conditionals `$(e.target).parent().is('a')` and `$(e.target).is('a')` if the concern is that the structure may change again.

Reply:
It will, but the time to traverse the DOM up to the root isn't going to be significant, so I'd favor code robustness over minimizing operations here. As a general rule, I'd say usage of `parent`, `children`, `sibling` and other functions that rely on DOM structure is a code smell. `closest` is still a little smelly, but less so.

Granted, we're not likely to do that much futzing with the markup inside of an anchor tag, so this is on the low-risk side.